api/app/Models/ApplicantFilter.php

```diff
@@ -45,6 +46,10 @@ public function classifications(): BelongsToMany
     {
         return $this->belongsToMany(Classification::class, 'applicant_filter_classification');
     }
+    public function streams(): BelongsToMany
+    {
+        return $this->belongsToMany(PoolStream::class, 'applicant_filter_stream');
+    }
```

Comment:
This is just a new field on a model and not a relation between tables so this would be wrong I think

Reply:
oh yes.. will remove that